mysys/my_atomic_writes.c

```diff
@@ -112,7 +112,7 @@ static my_bool test_if_shannon_card_exists()
     char path[32];
     struct stat stat_buff;
 
-    sprintf(path, "/dev/df%c", dev_part);
+    snprintf(path, sizeof(path), "/dev/df%c", dev_part);
```

Comment:
This one isn't required, the longest the path will ever be is 8 characters plus the NUL.